packages/next/build/swc/crates/core/src/react_remove_properties.rs

```diff
@@ -0,0 +1,76 @@
+use regex::Regex;
+use serde::Deserialize;
+use swc_ecmascript::ast::*;
+use swc_ecmascript::visit::{noop_fold_type, Fold};
+
+#[derive(Clone, Debug, Deserialize)]
+#[serde(untagged)]
+pub enum Config {
+    All(bool),
+    WithOptions(Options),
+}
+
+impl Config {
+    pub fn truthy(&self) -> bool {
+        match self {
+            Config::All(b) => *b,
+            Config::WithOptions(_) => true,
+        }
+    }
+}
+
+#[derive(Clone, Debug, Deserialize)]
+pub struct Options {
+    #[serde(default)]
+    pub properties: Vec<String>,
+}
+
+struct RemoveProperties {
+    properties: Vec<Regex>,
+}
+
+impl RemoveProperties {
+    fn should_remove_property(&self, name: &[u8]) -> bool {
```

Comment:
I think `name: &str` would be better, as it avoids `std::str::from_utf8`.

Reply:
Fixed! Did not realize `JsWord::as_ref` gives a `&str`.